includes/Core/Authentication/Clients/OAuth_Client.php

```diff
@@ -658,7 +652,7 @@ public function get_proxy_setup_url( $access_code = '', $error_code = '' ) {
 		);
 
 		if ( 'missing_verification' === $error_code ) {
-			$query_args['verification_nonce'] = wp_create_nonce( 'googlesitekit_verification' );
+			$query_args['verification_nonce'] = wp_create_nonce( Site_Verification::ACTION_VERIFICATION );
```

Comment:
A bit unrelated, but I think given we put the filter in the `get_proxy_setup_url()` method as I mentioned above, we could put this logic here into `Site_Verification`. This would also spare us the reference to the module class in core code.

Reply:
Updated 👍